### ISSUE TITLE
Add an option to show in-line diffing (one text area)

### ISSUE THREAD

**Issue by yelsayd** [yebrahim/python-difflib-gui#4]
Need a switch to go from two text areas to one that shows in-line differences and vice-versa

**Comment by TheSandDoctor** [comment]
I agree that this would be an awesome idea!! Too bad salsisan is inactive :( @yebrahim

**Comment by yelsayd** [comment]
Yea I'm not able to maintain this currently, but I'll make sure to look at incoming PRs, so feel free to suggest features or make changes. :)
